Chinese/deserthell/OldStory/Cat2Dog/saying/说了什么.md

```diff
@@ -0,0 +1,2 @@
+﻿白胡子老头说:“如果不是看在你爸的份儿上，今天我是不会再帮你的。快！原地转三圈！[然后......](./Cat2Dog/saying/说了什么.md)”
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.